static/src/js/containers/GranuleFiltersContainer/GranuleFiltersContainer.jsx

```diff
@@ -13,6 +13,7 @@ import { getFocusedCollectionGranuleQuery } from '../../selectors/query'
 import { getFocusedCollectionMetadata } from '../../selectors/collectionMetadata'
 
 import GranuleFiltersForm from '../../components/GranuleFilters/GranuleFiltersForm'
+import { metricsGranuleFilter } from '../../middleware/metrics/actions'
 
 export const mapStateToProps = (state) => ({
```

Comment:
I don't think that ` temporal: state.query.collection.temporal` is actually being used anymore